samples/oauth_rails_vethealth/app/assets/javascripts/health_api.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
Dropping the coffeescript dependency would be nice. As a sample app, there's a lot of value in this being as easy to approach as possible.

Reply:
Those were generated by rails, but since I'm not using them I'll delete them